include/formaction.h

```diff
@@ -72,6 +72,7 @@ class formaction {
 
 
 		void start_bookmark_qna(const std::string& default_title, const std::string& default_url, const std::string& default_desc, const std::string& default_feed_title);
+		void open_unread_items_in_browser(std::shared_ptr<rss_feed> feed , bool markread);
```

Comment:
That's not what I had in mind, but I admit that I just do "high-level" thinking and leave details for you to sort out.

I asked to put this into `formaction*s*`, a new, separate file, specifically because I didn't want to put it in `formaction` class. `formaction` is for sutff shared across all kinds of forms, while this "open all tabs" business is specific to feed- and itemlist.

Of course, I overlooked the fact that the method needs to know maximum number of tabs to open, thus needs access to `configcontainer` object. Passing `cfg` is tedious, so the method has to go into a class that inherits from `view`.

So here's another idea (which you are welcome to argue with if you see any drawbacks in it): create a mixin class that derives from `formaction`, put the method there, then derive `feedlist_formaction` and `itemlist_formaction` from that mixin (virtually, because diamond problem). What do you think?